### PR TITLE
[Cofense-32] Contrib/CDS Cofense Intelligence V2 111

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- Updated the Docker image to: *demisto/python3:3.10.6.33415*.
- **Note**: Updated the following commands to match the exact indicator to be searched for.
    - ***url***
    - ***domain***
    - ***ip***
    - ***email***
    - ***file***

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

**Comment by kgal-pan** [comment]
[UT build failure](https://app.circleci.com/pipelines/github/demisto/content/187153/workflows/f99b01d3-4ea8-4054-a68b-731eb2ee78f0/jobs/561722?invite=true#step-108-19) blocked by https://github.com/demisto/content/pull/20875.

**Comment by kgal-pan** [comment]
@crestdatasystems - Please fill out [the contribution form](https://forms.gle/XDfxU4E61ZwEESSMA) so we can start review.

**Comment by kgal-pan** [comment]
> [UT build failure](https://app.circleci.com/pipelines/github/demisto/content/187153/workflows/f99b01d3-4ea8-4054-a68b-731eb2ee78f0/jobs/561722?invite=true#step-108-19) blocked by #20875.

@crestdatasystems - Please sync with `master`.

**Comment by crestdatasystems** [comment]
@kgal-pan this is under progress and we want final approval from the client so please keep this on hold for a few days.

**Comment by YuvHayun** [comment]
> @kgal-pan this is under progress and we want final approval from the client so please keep this on hold for a few days.

No problem, please let us know when it's ready.

**Comment by YuvHayun** [comment]
@crestdatasystems how's it going?
Let me know if you need any help.

**Comment by crestdatasystems** [comment]
> @crestdatasystems how's it going? Let me know if you need any help.

We get final approval from the client, so @YuvHayun can start review this PR

File: Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py
Comment (YuvHayun) [inline Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py:215]:
```suggestion
    block_type: str = BLOCK_TYPE_MAPPING.get(command)
```

File: Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py
Comment (YuvHayun) [inline Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py:260]:
```suggestion
    block_type = BLOCK_TYPE_MAPPING.get("ip")
```

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Hi @crestdatasystems, thank you for your contribution.
Please see my comments below.

File: Packs/CofenseIntelligenceV2/.secrets-ignore
Comment (crestdatasystems) [reply Packs/CofenseIntelligenceV2/.secrets-ignore:6]: Yes, there are no secrets in the code. In addition, to ensure that we have executed the `demisto-sdk secrets` command,

File: Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py
Comment (crestdatasystems) [reply Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py:215]: Okay we will do this change

File: Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py
Comment (crestdatasystems) [reply Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py:260]: Okay we will do this change

**Comment by crestdatasystems** [comment]
> requested

Addressed the suggested changes

File: Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py
Comment (YuvHayun) [inline Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py:215]:
```suggestion
    block_type: str = BLOCK_TYPE_MAPPING.get(command, "")
```
This should fix the current Mypy problem.

**Review by YuvHayun — APPROVED** [review]

**Pull request merged**
